content/docs/operating/getting_started/talaria.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Talaria
+sort_rank: 2
+---
+
+# Talaria
+
+# Installation
+-   [RPM](https://xmidt.io/download/#talaria)
+-   [Binary](https://xmidt.io/download/#talaria)
+-   Docker (Link TBC)
+
+# Configuration
+Refer to [configuration file](https://github.com/xmidt-org/talaria/blob/master/talaria.yaml)
+for how to configure talaria.
+
+## Fixed
+For fixed routing, the service block in talaria's configuration should look similar to the example below:
+
+```yaml
+service:
+  defaultScheme: http
+  fixed:
+    - http://<HOSTNAME>:6200
+```
+Where HOSTNAME is you DNS record, docker container, or ip address.
+
+_**NOTE**_: if you have domain or host certificates available we recommend
+always running the service (and all components in the service) in https mode.
+
+## Consul
+For a consul managed list of talarias and caduceuses, the service block in talaria's configuration should look similar to the example below:
+
+```yaml
+service:
+  defaultScheme: http
+  consul:
+    client:
+      address: "CONSUL_ADDRESS:8500"
+      scheme: "http"
+      waitTime: "30s"
+    disableGenerateID: true
+    vnodeCount: 211
+    watches:
+      -
+        service: "talaria"
+        tags:
+          - "dev"
+          - "docker"
+        passingOnly: true
+    registrations:
+      -
+        id: "HOSTNAME"
+        name: "talaria"
+        tags:
+          - "dev"
+          - "docker"
+          - "stage=dev"
+          - "flavor=docker"
+        address: "http://HOSTNAME"
+        scheme: "http"
+        port: 6210
+        checks:
+          -
+            checkID: "talaria-1:http"
+            http: "http://HOSTNAME:HEALTH_PORT/health"
+            interval: "30s"
+            deregisterCriticalServiceAfter: "70s"
+```
```

Comment:
Add explanations for variables?
- [ ] `CONSUL_ADDRESS`
- [ ] `HOSTNAME`
- [ ] `HEALTH_PORT`